tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,79 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Icon, Section, Stack } from "../components";
+import { Window } from "../layouts";
+
+interface CrewCreditsData {
+  groups: GroupBlockProps[]
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600}>
+      <Window.Content scrollable>
+        {data.groups?.map((group, index) =>
+          ((data.groups[index].crew.length > 0) && <GroupBlock key={index} group={group.group} crew={group.crew} />)
```

Comment:
> If using `group` as a label below, should probably name `title` or `label` instead on the individual `group` items in the data structure.

mordent please clarify this

Reply:
`data.groups` is the list, and each item in that list is a group. The name (title/label) of the group should be `title`, `label`, or `name` and not `group` as it is here. `group.group` is weird,